src/pages/Activity.tsx

```diff
@@ -0,0 +1,121 @@
+import cardSampleImage from '@/assets/CardSample.png';
+import { Card } from '@/components/common/card/Card';
+import Title from '@/components/common/title/Title';
+
+const miniStudyData = [
+  {
+    name: 'Docker & Kubernetes 심화 스터디',
+    linkUrl: 'https://youtube.com/watch?v=docker_k8s_tips789',
+    imageUrl: cardSampleImage,
+  },
+  {
+    name: 'Python 웹 크롤링 실습 스터디',
+    linkUrl: 'https://youtube.com/watch?v=python_scraping456',
+    imageUrl: cardSampleImage,
+  },
+  {
+    name: 'Spring Boot & JPA 스터디',
+    linkUrl: 'https://youtube.com/watch?v=springboot_jpa123',
+    imageUrl: cardSampleImage,
+  },
+  {
+    name: 'Docker & Kubernetes 심화 스터디',
+    linkUrl: 'https://youtube.com/watch?v=docker_k8s_tips789',
+    imageUrl: cardSampleImage,
+  },
+  {
+    name: 'Python 웹 크롤링 실습 스터디',
+    linkUrl: 'https://youtube.com/watch?v=python_scraping456',
+    imageUrl: cardSampleImage,
+  },
+  {
+    name: 'Spring Boot & JPA 스터디',
+    linkUrl: 'https://youtube.com/watch?v=springboot_jpa123',
+    imageUrl: cardSampleImage,
+  },
+];
+
+const jecttalkData = [
```

Comment:
```suggestion
const jectTalkData = [
```
이건 생각하기 나름 + 수정안해도 됨!(어차피 삭제될 부분)